### ISSUE TITLE
Cannot Clear All and Select All using keyboard on consent management page

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6530]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome, Safari, Firefox
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to MyAccount -> Consent Management page
- Press TAB  until the focus is on CLEAR ALL or SELECT ALL
- Press ENTER key

Nothing happens. Cannot clear all and select all using the keyboard.
Also tried with SPACEBAR key.

<img width="611" alt="Screen Shot 2020-02-13 at 11 03 31 AM" src="https://user-images.githubusercontent.com/45409515/74454429-e2a4f380-4e51-11ea-9b63-64a83e34c489.png">

**Comment by kacperknapik** [comment]
### Proposed solution
Changing this `<a>` element to a `<button>` element + some basic logic, should do the trick

**Comment by mszczepaniak** [comment]
Tested successfully.
